### ISSUE TITLE
Timestamp of files and folder inside archive does not take into account timezone

### ISSUE THREAD

**Issue by qu1ck** [openscopeproject/ZipROFS#7]
Not sure if it's the file system's responsibility to calculate timezones or if it should always return them in UTC but it's the zip file that has them not in UTC.
Needs some investigation.

**Comment by JuniorJPDJ** [comment]
As far as I know filesystems in linux are holding time in UTC, but I may be wrong.
It's probably ZIP file.

**Comment by qu1ck** [comment]
Looking at this further, something weird is going on.
1. Zip has no notion of timezones and it's timestamps are 4 bytes only (limited precision) so there is no way to know whether they were in utc or in local time and what was the local time on the machine where the zip was created. https://en.wikipedia.org/wiki/Zip_(file_format)#:~:text=As%20a%20result%2C%20the%20built,zone%20they%20were%20created%20in.
So treating timestamps as if they are in UTC is as good a guess as any.
2. I am observing 7-zip showing file mtime as 
`Modified: 2008-07-17 15:05:34`
while `stat` on ziprofs currently reports it as 
`Modify: 2008-07-18 02:05:34.000000000 -0700`
which makes it 11 hours ahead.
Given than I'm in PDT timezone (UTC-7) I have no idea where 11 hours are coming from.

So I tested this on another zip file I created just now and timestamps in 7zip and ziprofs match.
My guess is that for previous file 7zip shows timestamps from extra metadata and not the main date_time field of file header. Side evidence to this is that for "bad" file 7zip also shows accessed and created timestamps but doesn't show them for a file I created just now using `zip` in debian.

TLDR: probably nothing to do here unless we want to get into business of parsing extra metadata which is specific to each archiver, not really standardized and not directly supported by zipfile lib.
